jplag/src/main/java/de/jplag/reportingV2/jsonfactory/JsonFactory.java

```diff
@@ -0,0 +1,60 @@
+package de.jplag.reportingV2.jsonfactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import de.jplag.reportingV2.reportobject.model.ComparisonReport;
+import de.jplag.reportingV2.reportobject.model.JPlagReport;
+
+/**
+ * Factory class, responsible for creating Json strings and writing them to files.
+ */
+public class JsonFactory {
+
+    /**
+     * Uses Jackson to create Json Strings from JPlagReport object.
+     * @return A list, first element is Json String of Overview object. The rest elements are Json Strings of Comparison
+     * objects.
+     */
+    public static List<String> getJsonStrings(JPlagReport jPlagReport) {
+        List<String> jsonReports = new ArrayList<>();
+        try {
+            ObjectMapper mapper = new ObjectMapper();
+            jsonReports.add(mapper.writeValueAsString(jPlagReport.getOverviewReport()));
+            for (ComparisonReport comparisonReport : jPlagReport.getComparisons()) {
+                jsonReports.add(mapper.writeValueAsString(comparisonReport));
+            }
+        } catch (JsonProcessingException e) {
+            System.out.println("Error converting object to json " + e.getMessage());
+        }
+        return jsonReports;
+    }
+
+    /**
+     * Creates Json Files for the given JPlagReport and saves them in the given folder.
+     * @return A boolean, representing whether the process was successful.
+     */
+    public static boolean saveJsonFiles(JPlagReport jPlagReport, String folderPath) {
+        String sanitizedFolderPath = folderPath;
+        if (!folderPath.endsWith("/")) {
```

Comment:
You assume linux paths

Reply:
Maybe simply use [Path](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Path.html#relativize-java.nio.file.Path-)

Reply:
Agree on that, paths are currently platform dependant. I will fix this.